design-documents/testing/functional/versioning-and-backward-compatibility-policy.md

```diff
@@ -28,82 +31,75 @@ An approach of defining what each release should include was taken from [Semanti
-Changing any of this attribute will cause Backward Incompatible change.
-
-**Test entity:**
-  
-  |xPath|attribute|
-  |---|---|
-  |`/tests/test`|name|
-  |`/tests/test/<ACTION> ⃰`|stepKey|
-  |`/tests/test/before/<ACTION> ⃰`|stepKey|
-  |`/tests/test/after/<ACTION> ⃰`|stepKey| 
-
-**Action Group entity:**
-
-  |xPath|attribute|
-  |---|---|
-  |`/actionGroups/actionGroup`|name|
-  |`/actionGroups/actionGroup/arguments/argument`|name|
-  |`/actionGroups/actionGroup/arguments/argument`|type|
-  |`/actionGroups/actionGroup/<ACTION> ⃰`|stepKey|
-
-**Data entity:**
-  
-  |xPath|attribute|
-  |---|---|
-  |`/entities/entitie`|name|
-  |`/entities/entitie/data`|key|
-  |`/entities/entitie/array`|key|
-  |`/entities/entitie/var`|key|
-  |`/entities/entitie/requiredEntity`|type|
-
-**Metadata entity:**
-  
-  |xPath|attribute|
-  |---|---|
-  |`/operations/operation`|name|
-  |`/operations/operation/field`|key|
-  |`/operations/operation(/object)+`|key|
-  |`/operations/operation(/object)+/field`|key|
-  |`/operations/operation(/object)+/array`|key|
-  |`/operations/operation/array`|key|
-  |`/operations/operation/array/object`|key|
+## Version increase matrix
   
-**Page entity:**
-
-  |xPath|attribute|
-  |---|---|
-  |`/pages/page`|name|
-  |`/pages/page/section`|name|
-  
-**Section entity:**
-
-  |xPath|attribute|
-  |---|---|
-  |`/sections/section`|name|
-  |`/sections/section/element`|name|
+|Entity Type|Change|Version Increase|
+|---|---|---|
+|ActionGroup|`<actionGroup>` added|MINOR
+| |`<actionGroup>` removed|MAJOR
+| |`<actionGroup>` `<action>` added|MINOR
+| |`<actionGroup>` `<action>` removed|MAJOR
+| |`<actionGroup>` `<action>` type changed|PATCH
+| |`<actionGroup>` `<action>` attribute changed|PATCH
+| |`<actionGroup>` `<argument>` added|MAJOR
+| |`<actionGroup>` `<argument>` removed|MAJOR
+| |`<actionGroup>` `<argument>` changed|MAJOR
+|Data|`<entity>` added|MINOR
+| |`<entity>` removed|MAJOR
+| |`<entity>` `<array>` added|MINOR
+| |`<entity>` `<array>` removed|MAJOR
+| |`<entity>` `<array>` `<item>` removed|PATCH
+| |`<entity>` `<field>` added|MINOR
+| |`<entity>` `<field>` removed|MAJOR
+| |`<entity>` `<required-entity>` added|MAJOR
+| |`<entity>` `<required-entity>` removed|MAJOR
+| |`<entity>` `<var>` added|MAJOR
+| |`<entity>` `<var>` removed|MAJOR
+|Metadata|`<operation>` added|MINOR
+| |`<operation>` removed|MAJOR
+| |`<operation>` changed|PATCH
+|Page|`<page>` added|MINOR
+| |`<page>` removed|MAJOR
+| |`<page>` `<section>` added|MINOR
+| |`<page>` `<section>` removed|MAJOR
+|Section|`<section>` added|MINOR
+| |`<section>` removed|MAJOR
+| |`<section>` `<element>` added|MINOR
+| |`<section>` `<element>` removed|MAJOR
+| |`<section>` `<element>` changed|PATCH
+|Test|`<test>` added|MINOR
+| |`<test>` removed|MAJOR
+| |`<test>` (before/after) `<action>` added|MINOR
+| |`<test>` (before/after) `<action>` removed|MAJOR
+| |`<test>` (before/after) `<action>` changed|PATCH
+| |`<test>` (before/after) `<action>` type changed|PATCH
```

Comment:
Please clarify what is `type`

Reply:
`type` = `click` for example, I'll change wording to make it more clear.